dcs_deploy.py

```diff
@@ -483,10 +593,29 @@ def extract_resource(self, resource, extract_path = None, need_sudo = False):
         stop_event.set()
         l4t_animation_thread.join()
         return ret
+    
+    def prepare_sources_mandatory(self, ret=0):
+        # Regenerate ssh access in rootfs
+        print("Purging ssh keys, this part needs sudo privilegies:")
+        cmd_exec("/usr/bin/sudo /usr/bin/id > /dev/null")
+        ret += cmd_exec("sudo resources/purge_ssh_keys.sh " + 
+                 os.path.join(self.rootfs_extract_dir, 'home', 'dcs_user','.ssh'))
+        self.networking_setup()
         
+        # Set default power mode to max
+        print("Setting up power mode, this part needs sudo privilegies:")
+        cmd_exec("/usr/bin/sudo /usr/bin/id > /dev/null")
+        if self.args.target_device == 'xavier_nx':
+            ret += cmd_exec("sudo python scripts/set_default_powermode.py " +
+                            os.path.join(self.rootfs_extract_dir, 'etc', 'nvpmodel', 'nvpmodel_t194_p3668.conf') +
+                            " 8")
+        if self.args.target_device == 'orin_nx':
+            # TODO: find nvpmodel.conf file for Orin and it's specific high power mode
+            print('Default power mode for Orin NX is not implemented yet.')
```

Comment:
Do we want to leave it for now? This should be highlighted or the user should be prompted more aggressively.

Reply:
No I do not want to leave this. Please let's have a discussion about this, I want this to be functional in Orin as well. I just could not find the file myself.

Reply:
The relevant files for setting power modes:
orin nx 16gb - `nvpmodel_p3767_0000.conf`
orin nx 8gb - `nvpmodel_p3767_0001.conf`
orin nano nx 8gb - `nvpmodel_p3767_0003.conf`
orin nano nx 4 gb - `nvpmodel_p3767_0004.conf`

The file names are enough to implement it?
 @eMrazSVK

Reply:
yes, it should be enough. Can you give me the names, that are used in `config_db.json` for all mentioned devices? For example for Xavier NX the key is `xavier_nx`. What is it like for other devices? Especially how do we differentiate 4 vs 8 vs 16 GB versions?

Reply:
These are the keys: 

` "device": ["orin_nx","orin_nx_8gb","orin_nano_4gb","orin_nano_8gb"]`